spec/ewb/extensions/IEC61970/Base/Wires/BatteryControl.yaml

```diff
@@ -0,0 +1,18 @@
+name: BatteryControl
+description: Describes behaviour specific to controlling batteries.
+attributes:
+  - name: chargingRate
+    type: Double
```

Comment:
todo: Should link to appropriate units (as per description). i.e. [PerCent](https://zepben.github.io/evolve/docs/cim/ewb/IEC61970/Base/Domain/PerCent) in this case. This also applies to other units available in the `IEC61970/Base/Domain` package.

Reply:
Changed it up and went through the other file to link up what I think is correct.